### DISCUSSION TITLE
Multiple node support?

### DISCUSSION THREAD

**Discussion by settle** [keras-team/autokeras#1607] [Q&A]
@haifeng-jin and others, I read that multiple GPUs are supported via distributed_strategy using MirroredStrategy, but what about multiple node support, i.e., using MultiWorkerMirroredStrategy or something similar?  https://www.tensorflow.org/guide/distributed_training

**Comment by settle** [comment]
@haifeng-jin and @qingquansong, I saw that your "Automated Machine Learning in Action" MEAP book with Manning ([link](https://www.manning.com/books/automated-machine-learning-in-action)) a section on multi-GPU and a Jupyter notebook example ([link](https://github.com/datamllab/automl-in-action-notebooks/blob/master/8.2-Parallelization-on-multiple-GPUs.ipynb)).  Would one of you please comment regarding support for parallelization on multi-nodes?  It would be nice addition to the scaling section in your book too.

**Reply by haifeng-jin** [reply]
We haven't tested the multi-node support in autokeras. It is now only officially supported in KerasTuner.

**Reply by settle** [reply]
@haifeng-jin Thank you for pointing that out.  I think you're referring to Section 8.2.3 "Parallel tuning" with environment variables for tuner IDs, oracle IPs, and oracle ports, is that right, or something else entirely?
